hw/bsp/samd11_xplained/board.mk

```diff
@@ -0,0 +1,53 @@
+CFLAGS += \
+  -ffunction-sections \
+  -fdata-sections \
+  -Wl,--gc-sections \
+  -mthumb \
+  -mabi=aapcs-linux \
+  -mcpu=cortex-m0plus \
+  -nostdlib -nostartfiles \
+  -D__SAMD11D14AM__ \
+  -DCONF_DFLL_OVERWRITE_CALIBRATION=0 \
+  -DOSC32K_OVERWRITE_CALIBRATION=0 \
+  -DCFG_TUSB_MCU=OPT_MCU_SAMD21 \
```

Comment:
Ideally we should add a new `OPT_MCU_SAMD11`, but it is not an issue. I will do a follow up later on. There is not much differences.